### ISSUE TITLE
suggestion: make it aware of pnpm

### ISSUE THREAD

**Issue by mkvlrn-cm42** [micalevisk/create-nest#1]
Would like to use it with pnpm, as in `pnpm create nest` or `pnpx create-nest` having it use pnpm to install packages.

Edit: looks like it just uses npm in any case, even when using yarn. Which is fine, I can just remove the lock file and node_modules, and `yarn install` or `pnpm install`.

Not sure how complicated it would be to actually use the manager that called the script because I'm clueless about shell scripting, but it would be the ideal scenario.

**Comment by micalevisk** [comment]
I wonder if we can detect the package manager being used so we can use it here:

https://github.com/micalevisk/create-nest/blob/55f2264d3420dd3b32b39f8744c5169c1d400bc2/create-nestjs-from-scratch.sh#L13-L17

**Comment by micalevisk** [comment]
not sure if it will work in 100% of the cases but it worked here

I tested with:

```
npm init nest
npm create nest
npx create-nest

yarn create nest

pnpm create nest
pnpm dlx create-nest
```

this was released in v1.9.0
